hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -243,7 +246,25 @@ public List<HapiSpec> getSpecsInSuite() {
                 nestedContractCannotOverSendValue(),
                 depositMoreThanBalanceFailsGracefully(),
                 lowLevelEcrecCallBehavior(),
-                callsToSystemEntityNumsAreTreatedAsPrecompileCalls());
+                callsToSystemEntityNumsAreTreatedAsPrecompileCalls(),
+                hollowCreationFailsCleanly());
+    }
+
+    private HapiSpec hollowCreationFailsCleanly() {
```

Comment:
Do we also want a BDD test for insufficient gas, as seen elsewhere in this PR (e.g. at `HederaMessageCallProcessorTest` unit test)?